R/translate.R

```diff
@@ -60,15 +60,33 @@ deepl_translate <- function(path,
     formality,
     values = c("default", "more", "less", "prefer_more", "prefer_less")
   )
-
-  # protect Hugo shortcodes ----
   temp_markdown_file <- withr::local_tempfile()
   markdown_lines <- brio::read_lines(path)
+  temp_markdown_file <- withr::local_tempfile()
+
+  # protect $$ equations $$
+  dollars <- which(grepl("\\$\\$", markdown_lines))
```

Comment:
what if there is a line like this:

```md

very expensive! "$$$$$" she thought.

```

Shouldn't the dollar detection be more specific, `dollar <- which(markdown_lines == "$$")` or something like that.


Note that instead of `which(grepl(.))` it is better to use `grep(.)` directly. I need to follow this advice myself :joy_cat: #62

Reply:
the comment on using `tinkr::protect_math()` makes this suggestion useless.

Reply:
yes ok, you're right !